### PR TITLE
Fix accounts REST API with DER encoded public key

### LINKED ISSUE #2025: Account REST API doesn't support ED25519 DER encoded prefix
**Detailed Description**
The accounts REST API with the publickey parameter doesn't work if the provided public key has the ED25519 DER encoded prefix of `302a300506032b6570032100`. Omitting the prefix works. The data in the database is all lowercase. The code lowercases the input then passes it to a library to validate that returns it as uppercase if it's DER encoded. We need to lowercase the library output.

**Actual Behavior**
curl -s "https://mainnet.mirrornode.hedera.com/api/v1/accounts?account.publickey=302a300506032b6570032100fe61c4ef478a7e05d39030c55024da73bb5efcd154d2dd7f6a29fb57dd7eb370"
```json
{"accounts":[],"links":{"next":null}}
```

**Expected Behavior**
curl -s "https://mainnet.mirrornode.hedera.com/api/v1/accounts?account.publickey=fe61c4ef478a7e05d39030c55024da73bb5efcd154d2dd7f6a29fb57dd7eb370"
```json
{"accounts":[{"balance":{"timestamp":"1622130301.216444000","balance":0,"tokens":[]},"account":"0.0.269385","expiry_timestamp":null,"auto_renew_period":7776000,"key":{"_type":"ED25519","key":"fe61c4ef478a7e05d39030c55024da73bb5efcd154d2dd7f6a29fb57dd7eb370"},"deleted":false},{"balance":{"timestamp":"1622130301.216444000","balance":0,"tokens":[]},"account":"0.0.269386","expiry_timestamp":null,"auto_renew_period":7776000,"key":{"_type":"ED25519","key":"fe61c4ef478a7e05d39030c55024da73bb5efcd154d2dd7f6a29fb57dd7eb370"},"deleted":false},{"balance":{"timestamp":"1622130301.216444000","balance":0,"tokens":[]},"account":"0.0.269404","expiry_timestamp":null,"auto_renew_period":7776000,"key":{"_type":"ED25519","key":"fe61c4ef478a7e05d39030c55024da73bb5efcd154d2dd7f6a29fb57dd7eb370"},"deleted":false},{"balance":{"timestamp":"1622130301.216444000","balance":0,"tokens":[]},"account":"0.0.269552","expiry_timestamp":null,"auto_renew_period":7776000,"key":{"_type":"ED25519","key":"fe61c4ef478a7e05d39030c55024da73bb5efcd154d2dd7f6a29fb57dd7eb370"},"deleted":false},{"balance":{"timestamp":"1622130301.216444000","balance":0,"tokens":[]},"account":"0.0.269567","expiry_timestamp":null,"auto_renew_period":7776000,"key":{"_type":"ED25519","key":"fe61c4ef478a7e05d39030c55024da73bb5efcd154d2dd7f6a29fb57dd7eb370"},"deleted":false}],"links":{"next":null}}
```

**Environment:**
- OS: Ubuntu 18.04
- Version: v0.32.0

**Additional Context**

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #2025

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: hedera-mirror-rest/__tests__/specs/accounts-05-publickey.spec.json [hunk 1]
```diff
@@ -59,7 +59,14 @@
     "transactions": [],
     "cryptotransfers": []
   },
-  "url": "/api/v1/accounts?account.publickey=3c3d546321ff6f63d701d2ec5c277095874e19f4a235bee1e6bb19258bf362be",
+  "urls": [
```
Comment (Nana-EC) [inline hedera-mirror-rest/__tests__/specs/accounts-05-publickey.spec.json:62]: Can you update the description / extended description with the fact that you're capturing variation of the der case. So it's clear to someone looking at the spec file what you're doing here

**Review by Nana-EC — CHANGES_REQUESTED** [review]
One minor request, looks good otherwise

File: hedera-mirror-rest/__tests__/specs/accounts-05-publickey.spec.json (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-rest/__tests__/specs/accounts-05-publickey.spec.json:62]: Done

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**